docs/xks/operator-guide/github.md

```diff
@@ -98,13 +98,13 @@ This is to prevent unnecessary masking of { } in your logs which are in dictiona
 }
 ```
 
-Upload the entire json as your github secret.
-The workflow uses one secret per environment and we recommend that you follow our namestandard.
-The secret name the workflow use is **AZURE_CREDENTIALS_\<ENV\>**, for  example **AZURE_CREDENTIALS_DEV**
+Upload the entire JSON as your GitHub secret.
+The workflow uses one secret per environment and we recommend that you follow our naming standard.
+The secret name the workflow uses is **AZURE*CREDENTIALS*\<ENV\>**, for example **AZURE_CREDENTIALS_DEV**.
 
-To upload the secret to github you can use the github UI or you can use the [gh cli](https://github.com/cli/cli) to upload secrets to GitHub.
+To upload the secret to GitHub you can use the GitHub UI or you can use the [GitHub CLI](https://github.com/cli/cli) to upload secrets to GitHub.
```

Comment:
Mixed feelings about changing gh cli to GitHub CLI since gh is the binary name. I don't know what creates the least amount of confusion.

Reply:
I was actually thinking about this when making the change. My motivation for changing it was that we link a product (GH CLI) - this is not a "run these commands" section.